jdbc-adapter/virtualschema-jdbc-adapter/src/main/java/com/exasol/adapter/jdbc/KerberosUtils.java

```diff
@@ -33,37 +33,28 @@ public static void configKerberos(String user, String pass) throws Exception {
         krbDir.delete();
         krbDir.mkdir();
         krbDir.deleteOnExit();
-        String krbConfPath = writeKrbConf(krbDir, confKeytab[0]);
-        String keytabPath = writeKeytab(krbDir, confKeytab[1]);
+        String krbConfPath = writePath(krbDir, confKeytab[0], "krb_", ".conf");
+        String keytabPath = writePath(krbDir, confKeytab[1], "kt_", ".keytab");
         String jaasConfigPath = writeJaasConfig(krbDir, user, keytabPath);
         System.setProperty("java.security.auth.login.config", jaasConfigPath);
         System.setProperty("java.security.krb5.conf", krbConfPath);
         System.setProperty("javax.security.auth.useSubjectCredsOnly", "false");
     }
 
-    private static String writeKrbConf(File krbDir, String base64Conf) throws Exception {
-        File file = File.createTempFile("krb_", ".conf", krbDir);
+    private static String writePath(File krbDir, String confKeyTab, String prefix, String suffix) throws Exception {
```

Comment:
Add `final` to all parameters and method variables.

Reply:
done